data/sql/updates/db_world/2022_09_03_01.sql

```diff
@@ -0,0 +1,47 @@
+INSERT INTO `smart_scripts` VALUES (36540, 0, 6, 0, 27, 0, 100, 513, 0, 0, 0, 0, 0, 75, 81004, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Mountain Horse - On Passenger Boarded - Add Aura');
+INSERT INTO `smart_scripts` VALUES (36540, 0, 7, 8, 28, 0, 100, 513, 0, 0, 0, 0, 0, 41, 0, 0, 0, 0, 0, 0, 11, 36555, 8, 0, 0, 0, 0, 0, 'Mountain Horse - On Passenger Remove - Despawn creatures');
+INSERT INTO `smart_scripts` VALUES (36540, 0, 8, 0, 61, 0, 100, 513, 0, 0, 0, 0, 0, 28, 68940, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 'Mountain Horse - On Link - Remove Aura');
+INSERT INTO `smart_scripts` VALUES (36555, 0, 0, 1, 54, 0, 100, 513, 0, 0, 0, 0, 0, 64, 1, 0, 0, 0, 0, 0, 23, 0, 0, 0, 0, 0, 0, 0, 'Mountain Horse - On just Summoned - Store Target');
+INSERT INTO `smart_scripts` VALUES (36555, 0, 1, 0, 61, 0, 100, 513, 0, 0, 0, 0, 0, 88, 3655500, 3655503, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Mountain Horse - On Link - Run Random Script');
+INSERT INTO `smart_scripts` VALUES (36555, 0, 2, 3, 38, 0, 100, 513, 2, 2, 0, 0, 0, 33, 36555, 0, 0, 0, 0, 0, 23, 0, 0, 0, 0, 0, 0, 0, 'Mountain Horse - On Data Set - Cast Spell');
+INSERT INTO `smart_scripts` VALUES (36555, 0, 3, 4, 61, 0, 100, 512, 0, 0, 0, 0, 0, 28, 68940, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Mountain Horse - On Link - Remove Aura');
+INSERT INTO `smart_scripts` VALUES (36555, 0, 4, 0, 61, 0, 100, 512, 0, 0, 0, 0, 0, 41, 100, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Mountain Horse - On Link Despawn');
+INSERT INTO `smart_scripts` VALUES (36555, 0, 5, 0, 27, 0, 100, 513, 0, 0, 0, 0, 0, 100, 1, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 'Mountain Horse - Passenger Boarded - Send Target');
+INSERT INTO `smart_scripts` VALUES (365550, 0, 0, 0, 54, 0, 100, 1, 0, 0, 0, 0, 0, 80, 36555000, 2, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Rope Bunny - On Just Summoned - Run Script');
+INSERT INTO `smart_scripts` VALUES (3655500, 9, 0, 0, 0, 0, 100, 1, 0, 0, 0, 0, 0, 29, 1, 45, 0, 0, 0, 0, 23, 0, 0, 0, 0, 0, 0, 0, 'Script - Follow Owner');
+INSERT INTO `smart_scripts` VALUES (3655501, 9, 0, 0, 0, 0, 100, 1, 0, 0, 0, 0, 0, 29, 1, 180, 0, 0, 0, 0, 23, 0, 0, 0, 0, 0, 0, 0, 'Script - Follow Owner');
+INSERT INTO `smart_scripts` VALUES (3655502, 9, 0, 0, 0, 0, 100, 1, 0, 0, 0, 0, 0, 29, 1, 270, 0, 0, 0, 0, 23, 0, 0, 0, 0, 0, 0, 0, 'Script - Follow Owner');
+INSERT INTO `smart_scripts` VALUES (3655503, 9, 0, 0, 0, 0, 100, 1, 0, 0, 0, 0, 0, 29, 1, 90, 0, 0, 0, 0, 23, 0, 0, 0, 0, 0, 0, 0, 'Script - Follow Owner');
+INSERT INTO `smart_scripts` VALUES (36555000, 9, 0, 0, 0, 0, 100, 1, 500, 500, 0, 0, 0, 11, 68940, 0, 0, 0, 0, 0, 12, 1, 0, 0, 0, 0, 0, 0, 'Script - Cast Rope');
+DELETE FROM `spell_script_names` WHERE `spell_id` = 68903;
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES ('68903', 'spell_round_up_horse');
+INSERT INTO `vehicle_template_accessory` (`entry`, `accessory_entry`, `minion`, `summontype`, `summontimer`) VALUES ('36555', '365550', '1', '8', '0');
```

Comment:
EN: when creating a vehicle_template_accessory always create it with the description and seat_id because otherwise it will give error when loading the query because it does not have default value those 2 columns.
ES: cuando vaya a crear un vehicle_template_accessory siempre creelo con el description y seat_id porque sino lo hace dara error al cargar el query porque no tiene valor predeterminado esas 2 columnas